Expensify/App
# Supports big screens mobile devices for popover menu

### Details


Changes the way it renders the popover menu for iPad, for the same as Web and Desktop.


### Fixed Issues

$ https://github.com/Expensify/App/issues/5754

### Tests


1. Enter Expensify App with an iPad.
2. Press the FAB(Plus button).
3. The Popover menu should appear the way it appears in Web and Desktop.(Animated from the side instead of from bottom)

### QA Steps


1. Enter Expensify App with an iPad.
2. Press the FAB(Plus button).
3. The Popover menu should appear the way it appears in Web and Desktop.(Animated from the side instead of from bottom)

### Tested On


### Screenshots


#### Web

<img width="476" alt="Screen Shot 2021-11-10 at 17 06 28" src="https://user-images.githubusercontent.com/53409604/141218540-26fe3df8-fef3-4e88-a11b-a536881e250f.png">

#### Desktop

<img width="490" alt="Screen Shot 2021-11-10 at 17 05 24" src="https://user-images.githubusercontent.com/53409604/141218535-2e75f64d-f7f6-4c9c-94ed-b1172955f11d.png">

#### iOS


##### iPhone
![iPhone](https://user-images.githubusercontent.com/53409604/141218354-193e88d5-32e5-4607-9f47-92abe918e7fa.png)

##### iPad

https://user-images.githubusercontent.com/53409604/141218476-427ada44-36fc-4323-9478-e8d716ea426d.mp4


#### Android



![Android](https://user-images.githubusercontent.com/53409604/141218372-06603deb-4a16-429d-b1ff-d3fb269c962d.png)
